packages/firebase_ml/android/src/main/java/io/flutter/plugins/firebaseml/ModelManager.java

```diff
@@ -0,0 +1,132 @@
+package io.flutter.plugins.firebaseml;
+
+import android.os.Build;
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.ml.common.modeldownload.FirebaseModelDownloadConditions;
+import com.google.firebase.ml.common.modeldownload.FirebaseModelManager;
+import com.google.firebase.ml.custom.FirebaseCustomRemoteModel;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel.Result;
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ModelManager {
```

Comment:
nit: For a utility class (where all members are static), it is recommended to add a private constructor to prevent subclassing and instantiating. You can also mark the class as final as well.

https://screenshot.googleplex.com/5MaeRMfBPHx

Reply:
Done